odl/solvers/functional/default_functionals.py

```diff
@@ -2470,6 +2470,149 @@ def __repr__(self):
                                        self.gamma)
 
 
+class BregmanDistance(Functional):
+    """The Bregman distance functional.
+
+    Notes
+    -----
+    Given a functional :math:`f`, which has a (sub)gradient :math:`\partial f`,
+    and given a point :math:`y`, the Bregman distance functional :math:`D_f`
+    in a point :math:`x` is given by
+
+    .. math::
+        D_f(x) = f(x) - f(y) - \langle \partial f(y), x - y \\rangle.
+
+
+    References
+    ----------
+    Wikipedia article: https://en.wikipedia.org/wiki/Bregman_divergence
+
+    [Bur2016] Burger, M. *Bregman Distances in Inverse Problems and Partial
+    Differential Equation*. In: Advances in Mathematical Modeling, Optimization
+    and Optimal Control, 2016. p. 3-33. Arxiv version online at
+    https://arxiv.org/abs/1505.05191
+    """
+
+    def __init__(self, functional, point, subgradient_op=None):
+        """Initialize a new instance.
+
+        Parameters
+        ----------
+        functional : `Functional`
+            The functional on which to base the Bregman distance. If the
+            optional argument `subgradient_op` is not given, the functional
+            needs to implement `functional.gradient`.
+        point : element of ``functional.domain``
+            The point from which to define the Bregman distance
+        subgradient_op : `Operator`, optional
+            The operator that takes an element in `functional.domain` and
+            returns a subgradient of the functional in that point.
+            Default: ``functional.gradient``.
+
+        Examples
+        --------
+        Example of initializing the Bregman distance functional:
+
+        >>> space = odl.uniform_discr(0, 2, 14)
```

Comment:
14 is rather arbitrary, prefer 10 or so

Reply:
It works with 8, 9, 11, 12, 13, and 14 points... but with 10 points apparently `Bregman_dist(space.zero())` returns 1.9999999999999998 and therefore fails... is there a way to make "almost equal" in doc-tests?